File: ExcAlgoritmoProgramacao_JS/Lista_Bee/1038_basico.js
import { question } from "readline-sync";

function main() {
    const entrada = get_number();

    let codigo_quantidade = entrada.split(` `);
    let codigo = parseInt(codigo_quantidade[0]);
    let quantidade = parseInt(codigo_quantidade[1]);

    let total_pagar = Calcular_Conta(codigo, quantidade);

    print(`Total: R$ ${total_pagar}`);
}

function get_number() {
    return question();
}

function print(texto) {
    return console.log(texto);
}

function Calcular_Conta(num1, num2) {
    if (num1 === 1) {
        return (num2 * 4).toFixed(2);
    } else if (num1 === 2) {
        return (num2 * 4.5).toFixed(2);
    } else if (num1 === 3) {
        return (num2 * 5).toFixed(2);
    } else if (num1 === 4) {
        return (num2 * 2).toFixed(2);
    }

    return (num2 * 1.5).toFixed(2);
}

main();

File: Exercicios_ED_I/Bee/1038_basico.cpp
#include <stdio.h>
#include <iostream>
using namespace std;

main() {
    int codigo, quantidade;
    cin >> codigo >> quantidade;
    double total_Pagar = 0.00;

    if (codigo == 1) {
        total_Pagar = 4.00 * quantidade;
        printf("Total: R$ %.2lf\n", total_Pagar);
    } else if (codigo == 2) {
        total_Pagar = 4.50 * quantidade;
        printf("Total: R$ %.2lf\n", total_Pagar);
    } else if (codigo == 3) {
        total_Pagar = 5.00 * quantidade;
        printf("Total: R$ %.2lf\n", total_Pagar);
    } else if (codigo == 4) {
        total_Pagar = 2.00 * quantidade;
        printf("Total: R$ %.2lf\n", total_Pagar);
    } else if (codigo == 5) {
        total_Pagar = 1.50 * quantidade;
        printf("Total: R$ %.2lf\n", total_Pagar);
    }
}